docs/internal-documentation/maintenance/ReleaseProcess-release.md

```diff
@@ -299,9 +299,70 @@ The releases pages:
 Send emails to the mailing lists `zonemaster-users` and `zonemaster-announce`.
 
 
+## Appendix A on version number in Makefile.PL
+
+As described above, `Makefile.PL` of Zonemaster-CLI and Zonemaster-Backend,
+respectively, must be specified with the lowest supported version of
+Zonemaster::LDNS and Zonemaster::Engine, respectively. Zonemaster-Engine
+must have that of Zonemaster::LDNS.
+
+The versions of Zonemaster::LDNS and Zonemaster::Engine are defined in the
+format `vX.Y.Z` and it is important how this is written in `Makefile.PL`.
+
+For lowest risk of error follow the following steps:
+
+1. Expand the version number with leading zeros on second ("Y") and third
+   ("Z") level. Remove the leading `v` and the second and third dots.
+   E.g. `v2.1.2` = `2.001002`.
+2. Decide if the third level can be skipped, e.g. `v2.1.2` > `2.001`.
```

Comment:
Since we're using semantic versioning the policy should be to always restrict the version requirement to only include the major and minor numbers.

Reply:
I do not agree. We should normally set what we have tested to lower the risk of problem.